### DISCUSSION TITLE
Can the time scale output interval be smaller?

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2937] [Q&A] [answered]
If the experiment is defined, I found that the interval of pybamm's time output value ("Time[s]") is fixed at 60s. Is there a way to change this interval to 1s or 10s?

**Comment by brosaplanella** [comment, marked as answer]
You can do that defining a period in the experiment, either as an argument when you call the [Experiment class](https://pybamm.readthedocs.io/en/latest/source/api/experiments/experiment.html) or adding the period of each step in brackets (see [this example](https://colab.research.google.com/github/pybamm-team/PyBaMM/blob/main/examples/notebooks/Getting%20Started/Tutorial%205%20-%20Run%20experiments.ipynb)).

**Reply by RobinBluehub** [reply]
Thank you , the problem has been solved!
